.changeset/orange-rabbits-juggle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/react-hooks': minor
```

Comment:
ah maybe. Though it already got released due to a bad push to an unprotected `main` branch https://www.npmjs.com/package/@shopify/react-hooks 

Keeping it as-is to avoid version issues

Reply:
Even if the minor has been released when this PR goes in it will just do a patch bump so there shouldn't be an issue I don't think.